SimplyLinkedList.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
+
+from Link import Link
+
+class SimplyLinkedList(object) :
+	
+	def __init__(self, head=None):
+		self.head = head
+		
+	# get the head link
+	def get_head(self):
+		return self.head
+		
+	# insert elements at the front of the list
+	def add_first(self, data):
+		new_node = Link(data)
+		new_node.set_next(self.head)
+		self.head = new_node
+	
+	# calculate the size of the linked list
+	def size(self):
+		currentLink = self.head
+		counter = 0
+		while currentLink:
+			counter += 1
+			currentLink = currentLink.get_next()
+		return counter	
+
+	# remove some data from the linked list
+	def remove(self, data):
+		# get the current link
+		current = self.head
+		# get the previous link
+		previous = None
+		# save whether the falue has been found
+		found = False
+		
+		# while the link is not None and the value has not been found
+		while current and found is False:
```

Comment:
Instead of `found is False` you can use `not Found`, which is easier to understand.